### ISSUE TITLE
Graphite sink - cluster.cpu.usage_rate is 2x the sum what of each node reports.

### ISSUE THREAD

**Issue by jbehrends** [kubernetes/heapster#1516]
heapster-amd64:v1.3.0-beta.1

Graphite sink:

**kubernetes.cluster.cpu.usage_rate** is reporting a value double of **sumSeries(kubernetes.nodes.*.cpu.usage_rate)**

When comparing **.limit** and **.request** values the same way, they match.

**Comment by theairkit** [comment]
@jbehrends Hi, I'll see it!

**Comment by jsoriano** [comment]
In case it gives any clue... where I'm using the riemann sink to forward to graphite I also see this behaviour, `kubernetes.X.cluster.cpu.usage_rate` doubles any of the sums of nodes, pods or namespaces: 
* `sum(kubernetes.X.nodes.*.cpu.usage_rate)`
* `sum(kubernetes.X.nodes.*.pods.*.*.cpu.usage_rate)`
* `sum(kubernetes.X.namespaces.*.cpu.usage_rate)`

@DirectXMan12 could you find someone to try it with other sinks to see if it also happens?

The sink is not doing any operation with the data provided... I'm wondering if the problem could be in the cluster aggregator.

**Comment by theairkit** [comment]
@jbehrends @jsoriano 
Yes, looks like that is not any sink (graphite, riemann etc.) bug,
sink does not do any opertion with data, its only 'proxy' them...

You may create issue on heapster more generally, not 'sink-related'.

(Anyway, for now I'll investigating this behavior)

**Comment by theairkit** [comment]
@jbehrends @jsoriano 
Seems like problem in cluster aggregator, watch PR: https://github.com/kubernetes/heapster/pull/1529

I think, this issue may be closed

**Comment by jbehrends** [comment]
UPDATE

I just pulled down the latest build of this container and tested sine the referenced PR was merged:
gcr.io/google-containers/heapster:v1.3.0

The cluster cpu_usage rate is no longer 2x the sum of the nodes cpu_usage rate, but it's now lower!  Not by a large margin but enough to not look correct.  See screenshot below:

![image](https://cloud.githubusercontent.com/assets/1915783/24160562/71105608-0e1f-11e7-928a-d4938368d481.png)

The drop in cluster usage_rate is when I deployed the updated heapster version.
